conda.recipe/meta.yaml

```diff
@@ -20,7 +21,9 @@ source:
 requirements:
   build:
     - nodejs
+    - npm
```

Comment:
`npm` comes for free when you install `nodejs` (this is why you are adding cpcloud channel?), so no need to add it...